src/test/api/karate/externaldevices/createDeviceConfig.feature

```diff
@@ -31,7 +31,7 @@ Feature: Device Configuration Management (Create)
     And header Authorization = 'Bearer ' + authToken
     And request configuration
     When method post
-    Then status 400
+    Then status 201
```

Comment:
The Scenario description should be "Update an existing configuration" ?
And as it is an update I would expect a 200 status code

Reply:
You're right, I've renamed the scenario description.